Casks/n/notion-mail.rb

```diff
@@ -0,0 +1,25 @@
+cask "notion-mail" do
+  version :latest
+  sha256 :no_check
+
+  url "https://s3.us-west-2.amazonaws.com/mail-desktop-release.notion-static.com/Notion%2BMail-latest-mac-universal.dmg",
+      verified: "s3.us-west-2.amazonaws.com/mail-desktop-release.notion-static.com/"
```

Comment:
```suggestion
```suggestion
  url "https://mail-desktop-release.notion-static.com/Notion%20Mail-#{version}-universal.dmg",
       verified: "mail-desktop-release.notion-static.com/"